homeassistant/components/environment_canada/__init__.py

```diff
@@ -87,6 +89,17 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     return unload_ok
 
 
+def device_info(config_entry: ConfigEntry) -> DeviceInfo:
+    """Build and return the device info for EC."""
+    return DeviceInfo(
+        entry_type=DeviceEntryType.SERVICE,
+        identifiers={(DOMAIN, config_entry.title)},
```

Comment:
Config entry title isn't stable (it can be changed by the end-user).

```suggestion
        identifiers={(DOMAIN, config_entry.entry_id)},
```